packages/php-wasm/universal/src/lib/load-php-runtime.ts

```diff
@@ -160,6 +161,9 @@ export async function loadPHPRuntime(
 	PHPRuntime.originalExit = PHPRuntime._exit;
 
 	PHPRuntime._exit = function (code: number) {
+		if (PHPRuntime.outboundNetworkProxyServer) {
+			PHPRuntime.outboundNetworkProxyServer?.closeAllConnections();
```

Comment:
We also need to call the close() method on the server instance here. 
closeAllConnections closes all open connections to the server but it does not close the server itself.